draft-ietf-tls-tls13.md

```diff
@@ -2957,8 +2956,17 @@ the responses).
 
 
 ### Key and IV Update {#key-update}
+     enum { update_not_requested(0), update_requested(1), (255)
```

Comment:
Sure, I would be onboard with that. Maybe "final enum" by analogy to final
classes?

On Mon, Sep 5, 2016 at 11:25 AM, David Benjamin <notifications@github.com>
wrote:

> In draft-ietf-tls-tls13.md
> <https://github.com/tlswg/tls13-spec/pull/611#discussion_r77549456>:
>
> > @@ -2957,8 +2956,17 @@ the responses).
> >
> >
> >  ### Key and IV Update {#key-update}
> > +     enum { update_not_requested(0), update_requested(1), (255)
>
> Oh I meant that more suggesting that we should find a way to denote it in
> the enum.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tlswg/tls13-spec/pull/611/files/306ccd2eea84f2120b2e61ef3a675c9eacb5e2bb#r77549456>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABD1oRdQlV_vCqUejLqTyO46N9idomu0ks5qnF63gaJpZM4Jx_WT>
> .
>